src/main/java/com/easypost/model/PaymentMethod.java

```diff
@@ -47,15 +47,6 @@ public String getId() {
         return id;
     }
 
-    /**
-     * Set ID of this PaymentMethod object.
-     *
-     * @param id ID of this PaymentMethod.
-     */
```

Comment:
I know these setters are not necessary, but why are you deleting them in this PR?